src/main/java/uk/gov/hmcts/befta/player/BackEndFunctionalTestAutomationDSL.java

```diff
@@ -60,4 +60,9 @@
     // a call [<some spec about the overall call data>] will get the expected response as in [<some test data unique id>]
     // another call [<some spec about the overall call data>] will get the expected response as in [<some test data unique id>]
     void performAndVerifyTheExpectedResponseForAnApiCall(String testDataSpec, String testDataId) throws IOException;
+
+    // DSL Element:
+    // "a wait time of [<wait time in seconds>] seconds to allow [<some spec about the scenario context>]"
+    // "a wait is allowed for [<wait time in seconds>] seconds [<some spec about the scenario context>]"
```

Comment:
`// "a wait is allowed for [<wait time in seconds>] seconds [<some spec about the scenario context>]"`
better like
`// "a wait time is allowed for [<wait time in seconds>] seconds [<some spec about the scenario context to describe the purpose of waiting>]"`